src/hooks/useAccommodationsAndOffers.tsx/helpers.ts

```diff
@@ -191,7 +192,9 @@ export const accommodationEventTransform =
     return { ...accommodation, eventInfo };
   };
 
-export const getGroupMode = (roomCount: number | string): boolean => {
+export const getGroupMode = (roomCount: number | string | undefined): boolean => {
+  if (process.env.DISABLE_FEATURES) return false;
```

Comment:
Should be `REACT_APP_DISABLE_FEATURES`. Since only environment variables which have the prefix `REACT_APP_` get populated onto the `process.env` object.